### ISSUE TITLE
一些补充

### ISSUE THREAD

**Issue by hua-ops** [zhangjunzhao-00/DANN-Bearing-fault-diagnosis#2]
非常感谢作者提供的代码，帮我这个学术混子应付了老师的任务。
不过，在运行代码的过程中，发现存在一些小疑问以及建议，希望可以帮助作者优化程序：
1. DANN网络中，域分类网络的标签使用的是 one-hot 编码格式，那么其输出层的激活函数使用 softmax 是否会更好一些
2. 由于所有的标签都是 one-hot 编码格式，那么其 accuracy 应该使用 交叉熵准确率（tf.keras.metrics.CategoricalAccuracy()）, 否则其计算得到的 accuracy  值会很小
3. 在 DANN_Model.train() 函数中，得到了 x_both 和 domain_labels 后，因为数据是直接拼接的，所以为了更好的训练，防止出现过拟合等问题，建议将数据打乱
>
```python
# 对于训练域自适应的样本，需要打乱
idx = tf.random.shuffle(tf.range(x_both .shape[0]))
x_both = tf.gather(x_both , idx)
domain_labels = tf.gather(domain_labels, idx)
```
